aws/logs_monitoring/lambda_function.py

```diff
@@ -132,50 +138,112 @@ def __exit__(self, ex_type, ex_value, traceback):
             print("DatadogConnection exit: ", ex_type, ex_value, traceback)
 
 
+class DatadogHTTPClient(object):
+
+    _POST = "POST"
+    _HEADERS = {"Content-type": "application/json"}
+
+    def __init__(self, host, apiKey, timeout=1, max_retries=1, backoff=0.5):
```

Comment:
Is a max_retries of 1 a sane default?

Reply:
reused current logic, I think it's fine, WDYT ?